.github/workflows/database_backup.yml

```diff
@@ -0,0 +1,28 @@
+name: Backup Database to Azure Storage
+concurrency: build_and_deploy_main
+
+on:
+  workflow_dispatch:
+  # TODO: Uncomment after migration
+  # schedule: # 03:00 UTC
+  #   - cron: "0 3 * * *"
+
+jobs:
+  backup:
+    name: Backup AKS Database (production)
+    if: ${{ github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' }}
```

Comment:
Is that necessary? The only triggers are schedule and workflow_dispatch

Reply:
Removed!